codespell_lib/data/dictionary.txt

```diff
@@ -37236,6 +37340,7 @@ visiters->visitors
 visitng->visiting
 visivble->visible
 vissible->visible
+vist->visit
```

Comment:
Perhaps `vast`, `vest`, `mist`, `list`, `fist`, `gist`, `vista`?

Also `vists->visits` (plural)

Reply:
Indeed, added. I ended up with this hunk:

```
-visted->visited
-visting->visiting
+vist->visit, vast, vest, mist, list, fist, gist, vista,
+visted->visited, listed, vested,
+visting->visiting, listing,
 vistors->visitors
+vists->visits, lists, fists,
```